app/assets/stylesheets/partials/_layout.css.scss

```diff
@@ -0,0 +1,19 @@
+.main {
+  @include outer-container;
+}
+
+.wrapper { 
+  @include shift(1);
+  @include span-columns(10); 
+  
+  .form {
+    @include span-columns(8);
+    padding: 5px 0;
+
+    textarea {
```

Comment:
So yeah, this was a problem at my old job, where people would nest css rules like 10 deep and then they would be impossible to overwrite because they're so specific.

This rule works out to be equivalent to `.wrapper .form textarea`.

Could you do it with less? like `form textarea` or `.wrapper textarea` ?